src/definitions/behaviors/visibility.js

```diff
@@ -902,7 +902,7 @@ $.fn.visibility = function(parameters) {
               element.offset.top += $context.scrollTop() - $context.offset().top;
             }
             if(module.is.horizontallyScrollableContext()) {
-              element.offset.left += $context.scrollLeft - $context.offset().left;
+              element.offset.left += $context.scrollLeft() - $context.offset().left;
```

Comment:
Note) [$elem.scrollLeft is a function](https://api.jquery.com/scrollLeft/), so it requies `()` when invoked.